### DISCUSSION TITLE
Is there a package that have hooks or utils for react?

### DISCUSSION THREAD

**Discussion by MiniSuperDev** [ReactiveX/rxjs#7354] [Q&A]
Thanks

**Comment by jakovljevic-mladen** [comment]
Ben has two repositories: https://github.com/benlesh/rxjs-react-hooks and https://github.com/benlesh/rxooks

**Comment by JoelCodes** [comment]
Also, I took some hooks that I keep using in my projects and made them available as an NPM library. https://www.npmjs.com/package/rx-xtra.react-hooks
